packages/app-server-issuer/src/types/index.ts

```diff
@@ -1,9 +1,14 @@
-import { CkitProvider, internal, RcSupplyLockHelper } from '@ckitjs/ckit';
+import { CkitConfigKeys, CkitProvider, internal, RcSupplyLockHelper } from '@ckitjs/ckit';
 
 export interface ServerContext {
   ckitProvider: CkitProvider;
   txSigner: InstanceType<typeof internal['Secp256k1Signer']>;
   rcHelper: RcSupplyLockHelper;
+  exchangeSigners?: Partial<
+    {
+      [field in CkitConfigKeys]: InstanceType<typeof internal['Secp256k1Signer']>;
+    }
+  >;
```

Comment:
```suggestion
  exchangeSigner: internal.Secp256k1Signer;
```

why we need multi kinds of signer for `exchangeSigner`?

Reply:
Because that the acp builder in ckit doesn't allow the address of sender and reciever to be the same, for convenient, here I use multiple lockscripts to do this.